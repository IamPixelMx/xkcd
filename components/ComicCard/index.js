import ComicCard from './ComicCard';

export { ComicCard };
