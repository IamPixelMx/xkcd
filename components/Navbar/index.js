import Navbar from './Navbar';
import NavbarInput from './NavbarInput';
import NavbarItem from './NavbarItem';

export { Navbar, NavbarInput, NavbarItem };
