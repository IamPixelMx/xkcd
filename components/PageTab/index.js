import PageTab from './PageTab';

export { PageTab };
