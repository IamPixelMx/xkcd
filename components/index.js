import { ComicCard } from './ComicCard';
import { Footer } from './Footer';
import { Navbar, NavbarInput, NavbarItem } from './Navbar';
import { PageTab } from './PageTab';
import { Pagination } from './Pagination';

export { ComicCard, Footer, Navbar, NavbarInput, NavbarItem, PageTab, Pagination };
