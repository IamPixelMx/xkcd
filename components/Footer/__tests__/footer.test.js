import React from 'react';
import renderer from 'react-test-renderer';
import { Footer } from '../../PageTab/__tests__/node_modules/components';

describe('Footer', () => {
  const footerElement = renderer.create(<Footer />).toJSON();

  it('Renders Footer', () => {
    expect(footerElement).toMatchSnapshot();
  });
});
