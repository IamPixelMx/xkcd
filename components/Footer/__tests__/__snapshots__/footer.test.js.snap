// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Renders Footer 1`] = `
<footer
  className="footer"
>
  <div
    className="content has-text-centered"
  >
    <p>
      <strong>
        XKCD-Mark
      </strong>
       fue construida con ❤ para
      <img
        alt="resuelve-logo"
        src="/resuelve.png"
        width="85"
      />
      por 
      <a
        href="https://itzelenciso.com"
      >
         Itzel Enciso
      </a>
      .
    </p>
    <p>
      El proyecto se ha deployado con Vercel ▲ y el código vive en
      <a
        href="https://github.com/eriika19/xkcd"
      >
         GitHub Repo 
      </a>
       . ¡Disfruta!
    </p>
  </div>
</footer>
`;
